src/main/java/org/jabref/gui/entryeditor/EntryEditor.css

```diff
@@ -126,15 +126,14 @@
     -fx-fill: #7800A9 ;
     -fx-font-size: 1.2em;
     -fx-font-weight: bolder;
-
 }
 
 #citationsPane {
     -fx-padding: 0;
     -fx-background-color: -fx-control-inner-background;
 }
 
-#citationsPane * {
```

Comment:
I'm a little worried about: what if this wildcard would affect style of other nodes? I guess, you tested everything and it works?

Also, I'm wondering, it doesn't work without this wildcard? 

+ A little space between class name and bracket will look good